code/modules/lighting/lighting_verbs.dm

```diff
@@ -0,0 +1,102 @@
+#define CHECK_RIGHTS(PERM)     \
```

Comment:
Unnecessary. check_rights already displays an error message. If it doesn't by default, then there's a boolean in its params you can touch to make it do so.